corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -51,11 +51,16 @@
         $("#archive-form").submit(function() {
             document.getElementById('archive-form-btn').disabled=true;
             document.getElementById('archive-spinner').style.display='inline-block';
-            ga_track_event('Reports', 'Case History', 'Archive this form', {
-                'hitCallback': function () {
+
+            // _.after(2,...) means the callback will only be called after *both* analytics
+            // functions have finished.
+            var analyticsCallback = _.after(2, function() {
```

Comment:
@NoahCarnahan I don't think this works (or at least something in my local dev environment is causing this to hang forever, and switching out the `_.after` to just call the callback fixes it.

it could also be related to https://github.com/dimagi/commcare-hq/commit/d79551caa51a77eac4643c23b5b400b2932488f8 which seems to touch the code that's called and is very recent.

any thoughts?

cc @sravfeyn